docs/en/FACL.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: global
+title: Access Control List
+nickname: Access Control List
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+This document describes the Access Control List feature related to file and directory permissions in Alluxio. 
+
+1. [ACL Permission Model](#ACL Permission Model): Alluxio implements an ACL model similar to those found in Linux and HDFS. This section describes some background on what this permission model looks like. 
+1. [Command Line Interface](#Command Line Interface): The users can interact with Alluxio's ACL model by using the command line interface tools such as `setfacl` and `getfacl`. 
```

Comment:
This is similar to a table of contents, so we probably don't need the table of contents if we're going to write it out like this